Form/AbstractModifySettingsForm.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Craue\ConfigBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * @author Christian Raue <christian.raue@gmail.com>
+ * @copyright 2011-2015 Christian Raue
+ * @license http://opensource.org/licenses/mit-license.php MIT License
+ */
+abstract class AbstractModifySettingsForm extends AbstractType {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function buildForm(FormBuilderInterface $builder, array $options) {
+		$builder->add('settings', Kernel::VERSION_ID < 20800 ? 'collection' : 'Symfony\Component\Form\Extension\Core\Type\CollectionType', array(
+			'type' => Kernel::VERSION_ID < 20800 ? 'craue_config_setting' : 'Craue\ConfigBundle\Form\Type\SettingType',
```

Comment:
@craue getBlockPrefix is a method added for the implementation of the usage of class names to reference types (because many other places which were using the name previously cannot use class names for that because of restrictions, for instance Twig block names).
If the method disappears in the future, it means that the way to reference form types will have changed again (and it would only be in a new major version of Symfony, for which you would have to add support explicitly anyway)